tests/test_stdiomgr_base.py

```diff
@@ -468,8 +468,6 @@ def test_stdout_detached(convert_newlines):
         f = o.detach()
 
         assert isinstance(f, io.BufferedRandom)
-        assert f is o._buf
```

Comment:
these assertions are almost enough to warrant keeping a reference, as a very blunt sanity check.

but I think `o.buffer` and `i.tee.buffer` *before* the `.detach()` are the same object, and could be stored in a variable of the test method and used in these assertions after the `.detach()`.

I half expect that we will need to fiddle with the `._buf` when we tackle encoding.

Reply:
Overall, I'm +0 on removing the `._buf` reference -- I agree, it could easily turn out to be useful/necessary down the line. If we end up wanting to muck about with the underlying buffers in the real `sys.stdio` (cf #75), my hunch is we'd also face wanting to muck about with the buffers in the mocks, even if not for the same reasons.

So, in the end, I'll not merge this at this time. If we approach a 2.0 release and find that it's truly not useful, we can trim it out then.